lang/julia/julia.py

```diff
@@ -0,0 +1,115 @@
+from talon import Context, Module, actions, settings
+
+mod = Module()
+ctx = Context()
+ctx.matches = """
+tag: user.julia
+"""
+
+ctx.lists["user.code_common_function"] = {
+    "print": "print",
+}
+
+ctx.lists["user.code_type"] = {
+    "integer": "Int",
+}
+
+
+@ctx.action_class("user")
+class UserActions:
+    def code_operator_subscript():
+        actions.user.insert_between("[", "]")
+
+    def code_insert_null():
+        actions.auto_insert("nothing")
+
+    def code_state_if():
+        actions.auto_insert("if\nend")
```

Comment:
Other languages with non-brace block delimiters (e.g. ruby, vimscript) seem to not include the closing delimiter (i.e. end) in their control block commands. So I reckon we shouldn't here either.

Reply:
I think generally I'm okay with removing them, and I'll remove it from the lua pull request. But it does make me wonder if there should be a separate action that corresponds to the full set of block delimiters? It seems like there's enough languages structured that way and a lot of the time I know I'm going to want both anyways so it would reduce the requirement to rely on snippets in the editor or having to chain commands.